ui/prometheus-plugin/src/plugins/prometheus-time-series-query/time-series-query-model.ts

```diff
@@ -19,6 +19,7 @@ import { PrometheusDatasourceSelector, TemplateString } from '../../model';
  */
 export interface PrometheusTimeSeriesQuerySpec {
   query: TemplateString;
+  series_name_format?: string;
```

Comment:
Could you update as well the corresponding Cue schema available here: https://github.com/perses/perses/blob/main/schemas/queries/prometheus/query.cue#L21-L31

Reply:
Yeah definitely, I'll update this soon!

Reply:
So the one part that feels weird here is that it sounds like maybe we want this to be an option on all time series queries, not just Prometheus ones? Or am I reading that wrong?

Reply:
Updated the cue schema and see [comment here](https://github.com/perses/perses/pull/709#discussion_r1016733954) about the TimeSeriesQuery vs. PrometheusTimeSeriesQuery part, going to chat you with some questions about how moving it to TimeSeriesQuery would potentially work